src/Libraries/Microsoft.Extensions.AI.Abstractions/README.md

```diff
@@ -646,7 +625,7 @@ var tracerProvider = OpenTelemetry.Sdk.CreateTracerProviderBuilder()
 var generator = new EmbeddingGeneratorBuilder<string, Embedding<float>>(
         new SampleEmbeddingGenerator(new Uri("http://coolsite.ai"), "my-custom-model"))
     .UseDistributedCache(new MemoryDistributedCache(Options.Create(new MemoryDistributedCacheOptions())))
-    .UseOpenTelemetry(sourceName)
+    .UseOpenTelemetry(sourceName: sourceName)
```

Comment:
Has the label been added for emphasis or because a change of API has made the previous form ambiguous?

Reply:
It's no longer the first parameter.